### ISSUE TITLE
BMP Inspection Tab Sorting Order

### ISSUE THREAD

**Issue by mortynsky** [leeensminger/DelDOT-NPDES-Viewer#75]
The list of BMP inspections under the, "BMP Inspection" tab does not list the BMP inspections sequentially in order form oldest to the most recent inspection date.  This was an issue in the old viewer, and is something DelDOT has specifically asked to fix for the new viewer.

![image](https://user-images.githubusercontent.com/16919984/27649200-1292a7d8-5bff-11e7-93f0-8921797904a8.png)

**Comment by cstevenson1** [comment]
@KCI-Ablowers Yes, everyone (including DelDOT) wants it to be sorted automatically to make it more user friendly. Many users don't realize the list can be sorted and/or from a quick glance don't realize the list isn't chronological, which has led to several incorrect assumptions.

**Comment by KCI-Ablowers** [comment]
@cstevenson1 Understood, the list should load automatically in a descending (newest to oldest) order.

**Comment by cstevenson1** [comment]
@kanglu666 @KCI-Ablowers push to prod
